.travis.yml

```diff
@@ -287,6 +287,16 @@ jobs:
     after_success:
       - bash <(curl -s https://codecov.io/bash)
 
+  - stage: Check deprecated Kubernetes apiVersions
+    os: linux
+    script:
+      - set -e # fail the whole script whenever any command fails
+      - /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+      - export PATH=$PATH:/home/linuxbrew/.linuxbrew/bin/
+      - brew install FairwindsOps/tap/pluto
```

Comment:
Among the available [installation options](https://github.com/FairwindsOps/pluto#installation), I chose brew so we always fetch the latest version of pluto

Reply:
TBH, I'm not a big fan of `brew`.

Could you install it by downloading the binary directly from GitHub releases by downloading it with curl/wget from https://github.com/FairwindsOps/pluto/releases/download/v3.5.1/pluto_3.5.1_linux_amd64.tar.gz, and extracting it using `tar -zxvf`?

This will avoid having an additional dependency of brew, and we have better control of the pluto version that we use.